.github/workflows/ci-tests.yaml

```diff
@@ -0,0 +1,31 @@
+name: CI Tests
+on:
+  release:
+    types: [opened, reopened]
+  push:
+  pull_request:
+    types: [opened, reopened]
+
+jobs:
+  run:
+    runs-on: ${{ matrix.operating-system }}
+    strategy:
+      matrix:
+        operating-system: [ ubuntu-latest, macos-latest ]
+        php-versions: [ '7.2', '7.3', '7.4', '8.0' ]
+    name: PHP ${{ matrix.php-versions }} Test on ${{ matrix.operating-system }}
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Setup PHP
+        uses: shivammathur/setup-php@v2
+        with:
+          php-version: ${{ matrix.php-versions }}
+          tools: phpunit
+
+      - name: Install dependencies
+        run: composer --prefer-source -n install
```

Comment:
Hello,

What is the purpose of using the `--prefer-source` here,  `--prefer-dist` would not be better?

Reply:
I'm not @Ciloe, but I'm assuming it's a direct copy from the existing `.travis.yml`:

> ```
> install:
>   - composer --prefer-source -n install
> ```

Reply:
Ok, I see but, is it wise or not?

Reply:
Yes it's a simple copy of travis